gcloud/streaming/http_wrapper.py

```diff
@@ -157,7 +173,7 @@ def length(self):
         Returns:
           Response length (as int or long)
         """
-        def ProcessContentRange(content_range):
+        def _process_content_range(content_range):
```

Comment:
In this case, if we move it, I would prefer a free helper function:  there is no `self` being used.